src/components/demo-app.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react'
 
+import DemoStep0 from './demo-step-0'
```

Comment:
Totally a function of my poor way of denoting steps here, but we should be consistent about either numbers or descriptive names for these steps. I think either is fine for now.

Reply:
Totally. The numbered steps made sense for the (literally) step-by-step demo, but I think best to re-write to function names. Or (for now) function + step, e.g. `create-listing-step-1`.